src/components/Asset/AssetActions/Compute/FormComputeDataset.tsx

```diff
@@ -156,7 +188,7 @@ export default function FormStartCompute({
           !isValid ||
           !isBalanceSufficient ||
           !isAssetNetwork ||
-          algorithmConsumableStatus > 0
+          !selectedAlgorithmAsset?.accessDetails.isPurchasable
```

Comment:
here we can add another chain op to avoid posible issues. change `!selectedAlgorithmAsset?.accessDetails.isPurchasable` to `!selectedAlgorithmAsset?.accessDetails**?**.isPurchasable`